commands/getex.md

```diff
@@ -0,0 +1,24 @@
+Get the value of `key`. Unlike `GET`, this is a write command and would be rejected by replicas. Return values are similar as `GET.
```

Comment:
i'm not sure the reference to replicas is right here.
in theory replicas can be set to accept writes.
```suggestion
Get the value of `key`. Unlike `GET`, this is a write command. Return values are similar as `GET.
```